### PR TITLE
Add Instructions for Generating API Client Modules

### PR DESCRIPTION
**Pull request by averyniceday**
Added general instructions for updating API Client Modules. 
Made in conjunction with updates to GN Annotation Pipeline docs.

GN Annotation Pipeline PR here: https://github.com/genome-nexus/genome-nexus-annotation-pipeline/pull/131

### CODE REVIEW

**Review by ao508 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # genome-nexus-java-api-client
 Java API Client for [Genome Nexus](https://www.genomenexus.org)
 
+To generate the api client, run:
```
Comment (sheridancbio) [inline README.md]: "To generate the API client, based on the currently deployed genome nexus production server, run:"

File: README.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+```
+. scripts/generateApiClientModules.sh
+```
+
```
Comment (sheridancbio) [inline README.md:11]: "To generate the API client, based on a local deployment of genome nexus, run:"

Comment (sheridancbio) [reply README.md:11]: (maybe these can be small headers)

**Review by sheridancbio — APPROVED** [review]
👍

**Pull request merged**